src/js/components/covid19/awardSpendingAgency/AwardSpendingAgency.jsx

```diff
@@ -126,15 +126,18 @@ const AwardSpendingAgency = () => {
         <div className="body__content spending-by-agency">
             <DateNote />
             <h3 className="body__narrative">
-                <strong>Which federal agencies</strong> issued awards using funds from COVID-19 spending?
+                <strong>Which agencies</strong> issued awards using COVID-19 funds?
             </h3>
             <div className="body__narrative-description">
                 <p>
                     Federal agencies receive funding from Congress and they issue awards to recipients using those funds. In this section we show which agencies and sub-agencies have awarded funds in response to the COVID-19 pandemic, as well as a breakdown of their obligated and outlayed funds.
                 </p>
                 <ReadMore>
                     <p>
-                        <em>Please note that agencies without COVID-19 appropriated funds are not represented here. Additionally, award amounts do not include the Small Business Administration (SBA)&apos;s Paycheck Protection Program.</em>
+                        <em>Please note that agencies without COVID-19 appropriated funds are not represented here.</em>
+                    </p>
+                    <p>
+                    <em>Additionally, award amounts do not include the Small Business Administration (SBA)&apos;s Paycheck Protection Program.</em>
```

Comment:
Indentation here is causing a lint error: `140:1   error  Expected indentation of 24 spaces but found 20`

Reply:
Fixed and committed!